sdk/batch/Azure.Compute.Batch/README.md

```diff
@@ -0,0 +1,86 @@
+# Azure Compute.Batch client library for .NET
+
+Azure.Compute.Batch is a managed service that helps developers get secret simply and securely.
+
+Use the client library for to:
+
+* [Get secret](https://docs.microsoft.com/azure)
```

Comment:
This does not look correct.   The intent of this is a couple of bullet points that describe champion scenarios for using the library.

- [Example: Event Grid](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/eventgrid/Azure.Messaging.EventGrid.Namespaces#azure-event-grid-client-library-for-net)
- [Example: App Config](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/appconfiguration/Azure.Data.AppConfiguration#azure-app-configuration-client-library-for-net)

Reply:
Sorry about that, I have now update the Readme to be accurate for Azure batch